DotaStats/Services/NetworkService.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+protocol NetworkSeviceProtocol: AnyObject {
+    func proPlayers() -> [Player]
+    func playersByName(_ name: String) -> [Player]
+    func proMatches() -> [Match]
+    func matchById(_ id: Int) -> Match
+    func playerInfoById(_ id: Int) -> PlayerInfo
+    func matchInfoById(_ id: Int) -> MatchInfo
+}
+
+class NetworkService: NetworkSeviceProtocol {
```

Comment:
Если вы согласны переименоваться, то можно сделать так
```suggestion
class NetworkServiceImp: NetworkSevice {
```